pandas/_libs/tslibs/period.pyx

```diff
@@ -131,9 +142,7 @@ cdef int64_t get_daytime_conversion_factor(int from_index, int to_index) nogil:
         int col = max_value(from_index, to_index)
     # row or col < 6 means frequency strictly lower than Daily, which
     # do not use daytime_conversion_factors
-    if row < 6:
-        return 0
-    elif col < 6:
+    if row < 6 or col < 6:
```

Comment:
Doesn't this need brackets to be done in the correct order?

Reply:
I have reverted this, as I was doing it wrong before, I haven't looked at the possibility that row can be let's say 12 and col be 2.

Reverted in 208bc03